### PR TITLE
implement seller logic + validation

### PR DESCRIPTION
**Pull request by IrynaKolh**

### CODE REVIEW

File: app/models/pet.rb [hunk 1]
```diff
@@ -2,4 +2,12 @@ class Pet < ApplicationRecord
   belongs_to :user
   has_rich_text :description
   has_many_attached :image
```
Comment (dsinn) [inline app/models/pet.rb:4]: Nit: `has_many` is generally followed by a plural noun, which would be `:images` in this case.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -44,16 +44,38 @@
...
         </div>
       </nav>
-
+    </div>
       <p class="notice"><%= notice %></p>
       <p class="alert"><%= alert %></p>
-    <%= yield %>
-    </div>
+
+    <main class=".main-content">
```
Comment (dsinn) [inline app/views/layouts/application.html.erb:69]: The `.` prefix is for CSS class selectors, but it doesn't belong in HTML.

```suggestion
    <main class="main-content">
```

**Review by dsinn — COMMENTED** [review]
The code looks good to me so far ))) 👍🏼

File: app/controllers/pets_controller.rb [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+
 
   # GET /pets or /pets.json
   def index
     @pets = Pet.all
   end
 
+  def my_pets
+    @my_pets = Pet.where(user_id: current_user.id).order(created_at: :desc)
+    render :my_pets
```
[the remark this answers is not in the document]
Comment (IrynaKolh) [reply app/controllers/pets_controller.rb:14]: No, I have 2 tabs: all pets and my pets. First - for view, searching all animals, second - for editing/deleting if you are owner.

File: app/models/pet.rb (hunk 1 above)
Comment (IrynaKolh) [reply app/models/pet.rb:4]: I know, maybe late I will change

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (IrynaKolh) [reply app/views/layouts/application.html.erb:69]: Fixed. Thank you!

**Pull request merged**